dll/win32/imm32/imm.c

```diff
@@ -1795,11 +1795,40 @@ UINT WINAPI ImmGetDescriptionA(
   return len - 1;
 }
 
+#ifdef __REACTOS__ /* FIXME: Don't put on here */
```

Comment:
This should indeed go into an imm32_undoc.h header of some sort...

Reply:
OK.